### ISSUE TITLE
Time tracking is not working in some cases

### ISSUE THREAD

**Issue by MikeMaximus** [MikeMaximus/gbm#173]
I noticed recently that the total time of my individual sessions do not match the stored hourly total for a few games.  Some games i've been playing a lot of lately are way off.

I think this is related to a problem with the game object not being refreshed when the same game is played over multiple sessions in sequence.

I'm not sure how long this has been broken, it was likely introduced in v1.1.5 when game detection was overhauled.

**Comment by basxto** [comment]
> total time of my individual sessions do not match the stored hourly total

That’s the case since the introduction of sessions. Playing hours were already counted before sessions existed, they will also get out of sync if sessions get disabled for some time.

**Comment by MikeMaximus** [comment]
You're right, for some users those hours probably won't be in sync and it's not really an issue.  But I was suspicious because in my case these were new games I only started playing in the last month, they were really out of sync and I always keep session tracking on.

Turned out the game object wasn't being updated with the new total hours after each session because the list key was changed in v1.1.5.  This caused time to be lost over multiple sessions of the same game until the game list gets reloaded.
